app/src/main/java/com/kickstarter/ui/viewholders/ProjectSearchResultViewHolder.java

```diff
@@ -1,75 +1,104 @@
 package com.kickstarter.ui.viewholders;
 
-import android.content.Context;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
+import android.util.Pair;
 import android.view.View;
 import android.widget.ImageView;
 import android.widget.TextView;
 
-import com.kickstarter.KSApplication;
 import com.kickstarter.R;
 import com.kickstarter.libs.KSString;
 import com.kickstarter.libs.utils.ObjectUtils;
-import com.kickstarter.models.Photo;
 import com.kickstarter.models.Project;
+import com.kickstarter.viewmodels.ProjectSearchResultHolderViewModel;
 import com.squareup.picasso.Picasso;
 
-import javax.inject.Inject;
-
 import butterknife.Bind;
 import butterknife.BindString;
 import butterknife.ButterKnife;
 
-public final class ProjectSearchResultViewHolder extends KSViewHolder {
-  private Project project;
-  private final Delegate delegate;
+import static com.kickstarter.libs.rx.transformers.Transformers.observeForUI;
+
+public class ProjectSearchResultViewHolder extends KSViewHolder {
+  private final ProjectSearchResultHolderViewModel.ViewModel viewModel;
 
-  protected @Bind(R.id.creator_name_text_view) TextView creatorNameTextView;
-  protected @Bind(R.id.project_name_text_view) TextView projectNameTextView;
-  protected @Bind(R.id.project_image_view) ImageView projectImageView;
+  protected final Delegate delegate;
+  private final KSString ksString;
 
-  protected @BindString(R.string.search_by_creator) String byCreatorString;
+  @Bind(R.id.project_name_text_view) TextView projectNameTextView;
+  @Bind(R.id.project_image_view) ImageView projectImageView;
+  @Bind(R.id.project_stats_text_view_pct_complete_data) TextView projectStatsPctCompleteDataTextView;
+  @Bind(R.id.project_stats_text_view_pct_complete_string) TextView projectStatsPctCompleteStringTextView;
```

Comment:
just curious as to what `Pct` stands for? it's not a very intuitive abbreviation for me